exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "Students' gardens are assigned in alphabetical order.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
+  ],
+  "cases": [
+    {
+      "description": "partial garden tests",
+      "cases": [
+        {
+          "description": "test garden with single student",
```

Comment:
at this point, all these descriptions begin with "test". I think the word could get removed from all of them, since it doesn't seem to be adding information to say that this is a test (should be self-evidence from its presence in this file, right?)

Sorry, I don't have a place where this "principle" (if it can even be called one) is documented, nor do I know for sure that all other files in the repo don't have "test" in their description (well, that can be found with grep...). So let me know what you think about it.